test/test-cases/braid-design-system/app/sku.config.js

```diff
@@ -1,10 +1,7 @@
 const ListExternalsPlugin = require('../../../utils/ListExternalsWebpackPlugin');
 
 module.exports = {
-  sites: [
-    { name: 'seekAnz', host: 'dev.seek.com.au' },
-    { name: 'jobStreet', host: 'dev.jobstreet.com' },
-  ],
+  sites: [{ name: 'seekAnz' }, { name: 'jobStreet' }],
```

Comment:
Any specific reason for this change?

Reply:
Yes these hosts arent setup in CI (or even guaranteed to be setup on a dev env). Given the test doesnt exercise these explicitly it made sense to remove them.